warp/cairo-src/evm/uint256.cairo

```diff
@@ -110,7 +112,8 @@ func uint256_signextend{range_check_ptr}(a : Uint256, i : Uint256) -> (res : Uin
     return (a)
 end
 
-func uint256_byte{range_check_ptr}(a : Uint256, i : Uint256) -> (res : Uint256):
+func uint256_byte{range_check_ptr, bitwise_ptr : BitwiseBuiltin*}(a : Uint256, i : Uint256) -> (
+        res : Uint256):
     let (i, _) = uint256_mul(i, cast((8, 0), Uint256))
```

Comment:
In this method you are still using `felt_and` function (line 120)

Reply:
Yeah, I rolled back some changes and accidently put that back there. I'll fix it, but I think that #136 is more urgent